portfolio/pom.xml

```diff
@@ -34,7 +46,6 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <!-- TODO: set project ID. -->
```

Comment:
I know this is a deleted TODO, but get in the habit of writing your TODOs as 'TODO(oumontiel)' so other people collaborating with you will know whose responsibility that task is!

Reply:
Will keep in mind, for sure! This TODO was already in the pom.xml file, though (I did not create it, just deleted it).

Reply:
Ah makes sense!